modules/test/playwright/tests/roles-admin-web/roles.spec.ts

```diff
@@ -1873,3 +1874,26 @@ test(
 		).toBeVisible();
 	}
 );
+
+test(
```

Comment:
is it possible to add this in an already existing test just adding these asserts and add also this tag to it? otherwise it's fine